cooltt.opam

```diff
@@ -13,6 +13,7 @@ depends: [
   "ocaml" {>= "4.10.0"}
   "ppx_deriving" {>= "4.4.1"}
   "cmdliner" {>= "1.0"}
+  "containers" {>= "3.4"}
```

Comment:
If we are going to use containers I think we can drop/replace/revise MANY parts of cooltt. @jonsterling